src/components/BondingCurveChart.tsx

```diff
@@ -76,14 +76,16 @@ const BondingCurveChart = observer(({}) => {
     const {
         root: { tradingStore, tokenStore, configStore, datStore },
     } = useStores();
+    
+    const activeDATAddress = configStore.getDXDTokenAddress()
```

Comment:
I believe the DAT contract is the ERC20 token, but maybe we should make that clear here or somewhere what the connection is between "DAT Address" and "DXD Token Address"

Reply:
Yes, we have to create an issue for that, the naming is confusing, this PR is already too big for that I think.